src/Docfx.App/Helpers/DocumentBuilderWrapper.cs

```diff
@@ -134,11 +134,11 @@ private static List<DocumentBuildParameters> ConfigToParameter(BuildJsonConfig c
 
         if (config.PostProcessors != null)
         {
-            parameters.PostProcessors = config.PostProcessors.ToImmutableArray();
+            parameters.PostProcessors = [..config.PostProcessors];
```

Comment:
Personally, I though CodeFixer rule for [IDE0305](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0305) should not be applied unconditionally

Because it decrease readability.
And no perf improvement by using collection initializer syntax.
(See: `https://github.com/dotnet/roslyn/issues/70656`)